explorer/ui/src/components/Pages.ts

```diff
@@ -11,6 +11,7 @@ export default class Pages {
     '/accounts/:accountPublicKeyBase16/deploys';
   static readonly Deploy = '/deploys/:deployHashBase16';
   static readonly Search = '/search';
+  static readonly ConnectedPeers = '/connectedPeers';
```

Comment:
```suggestion
  static readonly ConnectedPeers = '/peers';
```

Let's try and avoid camelCase in URLs, use dashes if we must.

Reply:
Done.